app/screens/home/search/results/post_results.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo} from 'react';
+import {FlatList, ListRenderItemInfo} from 'react-native';
+
+import NoResultsWithTerm from '@components/no_results_with_term';
+import DateSeparator from '@components/post_list/date_separator';
+import PostWithChannelInfo from '@components/post_with_channel_info';
+import {Screens} from '@constants';
+import {getDateForDateLine, isDateLine, selectOrderedPosts} from '@utils/post_list';
+import {TabTypes} from '@utils/search';
+
+import type PostModel from '@typings/database/models/servers/post';
+
+type Props = {
+    currentTimezone: string;
+    isTimezoneEnabled: boolean;
+    posts: PostModel[];
+    scrollPaddingTop: number;
+    searchValue: string;
+}
+
+const PostResults = ({
+    currentTimezone,
+    isTimezoneEnabled,
+    posts,
+    scrollPaddingTop,
+    searchValue,
+}: Props) => {
+    const paddingTop = useMemo(() => ({paddingTop: scrollPaddingTop, flexGrow: 1}), [scrollPaddingTop]);
```

Comment:
this is also used in `file_results`, you could pass the memoized styled as a prop to both components instead of `scrollPaddingTop`, then it will be memoized in one place, consuming less memory

Reply:
good catch. thanks.